backend/app/utils/json/json_loader.py

```diff
@@ -0,0 +1,31 @@
+from app.utils.encodings.encodings_list import encoding_list
+from app.utils.models.wikipedia_record import WikipediaRecord
+import json as json
+from typing import Tuple, Dict 
+
+
+def load_data(file_path: str) -> Tuple[Dict, ...]:
```

Comment:
Use `pathlib`.

Reply:
I am using `__file__` and `pathlib` in json_loader too.